packages/rxtranscribe/src/operators/toGCP.js

```diff
@@ -0,0 +1,65 @@
+import { of, from, throwError, combineLatest, forkJoin, zip} from 'rxjs';
+import { mergeMap, concatMap, takeUntil, map, tap, merge, switchMap, bufferCount, bufferTime, mergeAll, concatAll} from 'rxjs/operators';
+import gcpSpeech from '@google-cloud/speech';
+// import bufferBetweenSilences from '../operators/bufferBetweenSilences';
+
+const toGCP = function toGCP({
+  googleCreds = process.env.GOOGLE_APPLICATION_CREDENTIALS,
+  stop$ = of(),
+  sampleRate
+}) {
+  return fileChunk$ => {
+    // Creates a client
+    const clientv1 = new gcpSpeech.SpeechClient();
+    // const clientv1p1beta1 = new gcpSpeech.v1p1beta1.SpeechClient();
+    const config = {
+      encoding: 'LINEAR16',
+      sampleRateHertz: sampleRate,
+      languageCode: 'en-US',
+      // audioChannelCount: 2,
+      // enableSeparateRecognitionPerChannel: true,
+    };
+    return fileChunk$.pipe(
+      mergeMap(fileChunk => (
+        !googleCreds
+        ? throwError(new Error('Google Application Credentials must be set'))
+        : of(fileChunk)
+      )),
```

Comment:
💡 I was doing this a lot too in my code but I realized later that something like this saves a few lines of code and accomplishes the same result:
```js
return fileChunk => {
  if (!googleCreds) return throwError(new Error('Google Application Credentials must be set'));
  // ... otherwise, run the normal pipeline
}
```

Reply:
I'm always down for more concise code. Good tip.